packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Actions.js

```diff
@@ -53,18 +54,18 @@ const Actions = ({
         test-id={`${blockClass}__customize-columns-modal--reset`}
       >
         {resetToDefaultLabel}
-      </Button>
+      </Button> */}
     </div>
   );
 };
 
-const resetToOriginal = (columnDefinitions, originalColumnDefinitions) => {
-  const keyedDefs = keyBy(columnDefinitions, 'id');
-  return originalColumnDefinitions.map((colDef) => ({
-    ...keyedDefs[colDef.id],
-    isVisible: isColumnVisible(colDef),
-  }));
-};
+// const resetToOriginal = (columnDefinitions, originalColumnDefinitions) => {
+//   const keyedDefs = keyBy(columnDefinitions, 'id');
+//   return originalColumnDefinitions.map((colDef) => ({
+//     ...keyedDefs[colDef.id],
+//     isVisible: isColumnVisible(colDef),
+//   }));
+// };
```

Comment:
commented out code should really be cleaned out (there are of course exceptions).